src/PostProcess/RenderPipeline/Pipelines/babylon.defaultRenderingPipeline.ts

```diff
@@ -286,7 +286,7 @@
             this._reset();
 
             if(this.depthOfFieldEnabled){
-                this.depthOfField = new DepthOfFieldEffect(this._scene, this._depthOfFieldBlurLevel, this._defaultPipelineTextureType);
+                this.depthOfField = new DepthOfFieldEffect(this._scene, this._cameras[0], this._depthOfFieldBlurLevel, this._defaultPipelineTextureType);

```

Comment:
I think we should  not force cameras[0] but use the pipeline mechanism about active cameras

Reply:
If the scene has multiple cameras rendering at the same time, I dont want the second camera to get the firsts depth map.